transaction-status/src/parse_instruction.rs

```diff
@@ -31,6 +31,24 @@ pub enum ParseInstructionError {
 
     #[error("Program not parsable")]
     ProgramNotParsable,
+
+    #[error("Serde json error")]
```

Comment:
Is this exposed to the client? Maybe we could say "Internal error, please report"